pages/whom-we-are-hiring.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Who are we hiring?
```

Comment:
You're right; I missed the title entirely!

I personally don't mind the conceit, but I acknowledge it might induce a second, and not entirely pleasing, glance to the urchins whom we seek to hire.